src/main/kotlin/pl/diet/company/dietapp/controller/ProductController.kt

```diff
@@ -9,8 +9,11 @@ import pl.diet.company.dietapp.service.ProductQuery
 
 import pl.diet.company.dietapp.service.ProductReader
 
+import org.springframework.web.bind.annotation.*
```

Comment:
Please rebase this branch with current master version.

Reply:
ok